CHANGELOG.md

```diff
@@ -1,7 +1,109 @@
 # CHANGELOG
 
+## 4.5.00
+
+[Full Changelog](https://github.com/kokkos/kokkos/compare/4.4.01...4.5.00)
+
+### Features
+
+* Introduce new `SequentialHostInit` view allocation property [\#7229](https://github.com/kokkos/kokkos/pull/7229) (backported in 4.4.01)
+* Support building with Run-Time Type Information (RTTI) disabled
+* Add `KOKKOS_RELOCATABLE_FUNCTION` [\#5993](https://github.com/kokkos/kokkos/pull/5993)
+
+### Backend and Architecture Enhancements
+
+#### CUDA
+
+* Adding occupancy tuning for CUDA architectures [\#6788](https://github.com/kokkos/kokkos/pull/6788)
+* Add support for CUDA unified memory architectures, e.g.,  Grace Hopper [\#6823](https://github.com/kokkos/kokkos/pull/6823)
+* By default disable `cudaMallocAsync` (i.e., revert the change made in version 4.2) [\#7353](https://github.com/kokkos/kokkos/pull/7353)
+
+#### HIP
+
+* Add support for AMD Phoenix APUs with Radeon 740M/760M/780M/880M/890M [\#7162](https://github.com/kokkos/kokkos/pull/7162)
+* Update maximum waves per CU values for consumer card [\#7347](https://github.com/kokkos/kokkos/pull/7347)
+* Fix compilation of global launch for graph nodes [\#7365](https://github.com/kokkos/kokkos/pull/7365)
+* Check that Kokkos is running on the architecture it was compiled for [\#7379](https://github.com/kokkos/kokkos/pull/7379)
+* Add opt-in option to use `hipMallocAsync` instead of `hipMalloc` [\#7324](https://github.com/kokkos/kokkos/pull/7324)
+* Introduce new architecture flag `AMD_GFX942_APU` for MI300A [\#7462](https://github.com/kokkos/kokkos/pull/7462)
+
+#### SYCL
+
+* Move the `SYCL` backend out of the `Experimental` namespace [\#7171](https://github.com/kokkos/kokkos/pull/7171)
+* Introduce `KOKKOS_ENABLE_SYCL_RELOCATABLE_DEVICE_CODE` as CMake option [\#5993](https://github.com/kokkos/kokkos/pull/5993)
+
+#### OpenMPTarget
+
+Note: empty for now, delete section if none.
+
+#### OpenACC
+
+* Update CMake/make files to compile the OpenACC backend using Clacc [\#7198](https://github.com/kokkos/kokkos/pull/7198)
+* Fix OpenACC `MDRangePolicy` errors with `BasicView` [\#7425](https://github.com/kokkos/kokkos/pull/7425)
```

Comment:
Need to rephrase this one.  Should not be mentioning "BasicView".
I am tempted to say merge with the entry above and just say something like adding support for Clacc

Reply:
Why not just mentioning "Fix OpenACC `MDRangePolicy` errors"? Besides, this PR is not related to Clacc.

Reply:
This was a toolchain bug, essentially we got UB when using collapse clauses and we implemented a workaround